src/main/java/software/amazon/awssdk/crt/io/TlsContextOptions.java

```diff
@@ -136,6 +137,24 @@ public void initMTLSFromPath(String certificatePath, String privateKeyPath) {
         tlsContextOptionsInitMTLSFromPath(getNativeHandle(), certificatePath, privateKeyPath);
     }
 
+    /**
+     * Sets the certificate/key pair that identifies this TLS host. Must be in
+     * PEM format.
+     * 
+     * @param certificate PEM armored certificate
+     * @param privateKey  PEM armored private key
+     * @throws IllegalArgumentException If the certificate or privateKey are not in PEM format or if they contain chains
+     */
+    public void initMTLS(String certificate, String privateKey) throws IllegalArgumentException {
+        certificate = PemUtils.removeInvalidPemChars(certificate);
+        certificate = PemUtils.cleanUpPem(certificate);
```

Comment:
If you're cleaning up PEM's be default, I think there should be a check to see if the PEM changed after being cleaned up, and a warning logged if it did change.